QuickNoteVault.API/QuickNoteVault.DAL/UOW/UnitOfWork.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Storage;
+using QuickNoteVault.DAL.Repository;
+
+namespace QuickNoteVault.DAL.UOW
+{
+    public class UnitOfWork : IUnitOfWork
+    {
+        private readonly DbContext _context;
+        private readonly IDictionary<Type, object> _repositories = new Dictionary<Type, object>();
```

Comment:
Drop here `new Dictionary<Type, object>();`. Initialize repositories in constructor